charts/falco/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ numbering uses [semantic versioning](http://semver.org).
 ## v4.4.2
 
 * fix wrong check in pod template where `existingSecret` was used instead of `existingClientSecret`
+* Added a `labels` field in the controller to provide extra labeling for the daemonset/deployment
```

Comment:
The changelog has been mixed with the previous changes in the rebase process. Please add a new section for version: `3.3.3`